api.py

```diff
@@ -66,24 +69,28 @@ def layout(bundle, algo, scale=500.0):
 
 
 def process_bundle(bundle):
-    if len(bundle) > 1 and 'services' not in bundle:
-        raise BundleFormatException('This has multiple deployments')
-
-    if 'services' not in bundle:
-        if 'services' not in next(iter(bundle.values())):
-            raise BundleFormatException("This probably isn't a bundle...")
+    # In Juju 2.0 'services' is replaced by 'applications'
+    no_services = 'services' not in bundle and 'applications' not in bundle
+    if len(bundle) > 1 and no_services:
+        raise BundleFormatException('This bundle has multiple deployments.')
 
+    if no_services:
         bundle = next(iter(bundle.values()))
+        no_services = 'services' not in bundle and 'applications' not in bundle
+        if no_services:
+            raise BundleFormatException('This bundle has no applications.')
```

Comment:
This seems weird, we check `if no_services` but then redefine it again and then raise an exception?